sky/templates/aws-ray.yml.j2

```diff
@@ -75,10 +75,16 @@ available_node_types:
             sudo: ALL=(ALL) NOPASSWD:ALL
             ssh_authorized_keys:
               - skypilot:ssh_public_key_content
-        bootcmd:
-          - echo 'APT::Periodic::Enable "0";' > /etc/apt/apt.conf.d/10cloudinit-disable
-          - apt-get -y purge update-notifier-common ubuntu-release-upgrader-core landscape-common unattended-upgrades
-          - echo "Removed APT" | systemd-cat
+        write_files:
```

Comment:
Nice! Just a note for the future: some ulimit command in our setup_commands can possibly be moved to cloud-init time as well? E.g., https://github.com/lework/kainstall/blob/7a59178afe187f05cc4d78ce63947589f69fc9bd/kainstall-ubuntu.sh#L351-L376